sql/mysql/inspect.go

```diff
@@ -648,7 +648,7 @@ SELECT
 	t1.CREATE_OPTIONS
 FROM
 	INFORMATION_SCHEMA.TABLES AS t1
-	JOIN INFORMATION_SCHEMA.COLLATIONS AS t2
+	LEFT JOIN INFORMATION_SCHEMA.COLLATIONS AS t2
```

Comment:
Why is a left join needed?

Reply:
Because in TiDB you might not have collate information (see point 3 in my issue [here](https://github.com/pingcap/tidb/issues/32411))